widget/embedded/src/components/TokenList/TokenList.tsx

```diff
@@ -112,151 +117,146 @@ export function TokenList(props: PropTypes) {
     setTokens(list.slice(0, PAGE_SIZE));
   }, [list.length, selectedBlockchain]);
 
-  const renderList = () => {
-    if (!tokens.length && !!searchedFor) {
-      return (
-        <>
-          <Divider size={32} />
-          <NotFound
-            title={i18n.t('No results found')}
-            description={i18n.t('Try using different keywords')}
-          />
-        </>
-      );
-    }
+  if (!tokens.length && !!searchedFor) {
     return (
-      <VirtualizedList
-        endReached={hasNextPage ? loadNextPage : undefined}
-        itemContent={(index) => {
-          const token = tokens[index];
-          const tokenBalance = formatBalance(getBalanceFor(token));
-          const address = token.address || '';
-          const blockchain = blockchains.find(
-            (blockchain) => blockchain.name === token.blockchain
-          ) as BlockchainMeta;
-          const colors = createTintsAndShades(blockchain.color, 'main');
-          const customCssForTag = {
-            $$color: colors.main150,
-            [`.${darkTheme} &`]: {
-              $$color: colors.main750,
-            },
-            backgroundColor: '$$color',
-          };
-
-          const customCssForTagTitle = {
-            $$color: colors.main750,
-            [`.${darkTheme} &`]: {
+      <List as="ul">
+        <Divider size={32} />
+        <NotFound
+          title={i18n.t('No results found')}
+          description={i18n.t('Try using different keywords')}
+        />
+      </List>
+    );
+  }
+  return fetchStatus === 'loading' ? (
+    <LoadingTokenList size={PAGE_SIZE} />
+  ) : (
+    fetchStatus === 'success' && (
+      <List as="ul">
```

Comment:
'ul' element should only contain 'li' elements.

Reply:
This change has already been made and isn't related to this PR. Additionally, it's possible that modifying this part of VirtualizedList may not work correctly.